src/Configurator/EnvConfigurator.php

```diff
@@ -51,6 +51,12 @@ private function configureEnvDist(Recipe $recipe, $vars, bool $update)
             foreach ($vars as $key => $value) {
                 $value = $this->evaluateValue($value);
                 if ('#' === $key[0] && is_numeric(substr($key, 1))) {
+                    if ('' === $value) {
+                        $data .= "#\n";
+
+                        continue;
```

Comment:
Can we remove that "continue" and do an else to avoid the double-continue? Sorry for the nitpick

Reply:
sure, no problem